bayes/inference_problem.py

```diff
@@ -61,6 +64,72 @@ def add_noise_model(self, noise_model, key=None):
         self._noise_models[key] = noise_model
         return key
 
+    def set_parameter_prior(self, latent_name, dist_or_mean, sd=None):
```

Comment:
I would not mix the interface with either providing parameters or directly a distribution. I would suggest either using the kwargs feature in python, so that users directly provide the relevant properties of a scipy stats or always require the user to provide a scipy distribution.